src/components/Form/Form.js

```diff
@@ -0,0 +1,39 @@
+import  React, { Component } from 'react'
+import './Form.css'
+
+class Form extends Component {
+  constructor() {
+    super()
+    this.state = {
+      value: ''
+    }
+  }
+
+  handleSubmit = (event) => {
+    event.preventDefault()
+    this.props.searchByTitle(this.state.value)
+  }
+
+  handleChange = (event) => {
+    this.setState({value: event.target.value})
+  }
+
+render() {
+    return (
+      <form  className='search-body'onSubmit={this.handleSubmit}>
+        <label>
+          Search by title or enter your minimum desired movie rating:
+          <input 
+          type="text" 
+          name="name" 
+          className='input-field'
+          value={this.state.value} onChange={this.handleChange}
+          />
+        </label>
+        <button class='search-button'>RT</button>
```

Comment:
Should add a "clear search" button if we have time. Also, I'm not actually sure what RT means? Return?